### PR TITLE
dia.Cell: prop(), removeProp() should accept path defined as an array

### PR DESCRIPTION
**Pull request by vtalas**
ready for review

### CODE REVIEW

File: src/joint.dia.cell.js [hunk 1]
```diff
@@ -504,7 +518,25 @@ joint.dia.Cell = Backbone.Model.extend({
     removeAttr: function(path, opt) {
 
         if (_.isArray(path)) {
-            _.each(path, function(p) { this.removeAttr(p, opt); }, this);
+
+            var clone = path.slice();
```
Comment (kumilingus) [inline src/joint.dia.cell.js]: Do you have any special reason why you do not use `['attrs'].concat(path)` here? It's not slower (https://jsperf.com/array-unshift-vs-prepend/5) and easier to read IMO.

File: src/joint.dia.cell.js [hunk 2]
```diff
@@ -504,7 +518,25 @@ joint.dia.Cell = Backbone.Model.extend({
...
+            clone.unshift('attrs');
+            return this.removeProp(clone);
+        }
+
+        return this.removeProp('attrs/' + path, opt);
+    },
+
+    removeAttrs: function(path, opt) {
```
Comment (kumilingus) [inline src/joint.dia.cell.js]: The method supposed to remove multiple attributes i.e. `removeAttrs(paths, opt)`.  If `paths` is a string do not remove single attribute. That's why `removeAttr()` is there.

File: src/joint.dia.cell.js (hunk 1 above)
Comment (vtalas) [inline src/joint.dia.cell.js]: good point, totally agreed, I'll change it in method `attrs` as well

File: src/joint.dia.cell.js (hunk 2 above)
Comment (vtalas) [inline src/joint.dia.cell.js]: ok, it will throw an error path is not an array, also `path` should be `attrs` instead

**Pull request merged**